scripts/west_commands/tests/test_pyocd.py

```diff
@@ -37,21 +38,21 @@
     'dev_id': TEST_DEV_ID,
     'frequency': TEST_FREQUENCY,
     'daparg': TEST_DAPARG,
-    'tool_opt': [TEST_TOOL_OPT],
+    'tool_opt': TEST_TOOL_OPTS
 }
 
 TEST_DEF_KWARGS = {}
 
-TEST_ALL_PARAMS = (['--target', TEST_TARGET,
-                    '--daparg', TEST_DAPARG,
-                    '--pyocd', TEST_PYOCD] +
-                   ['--flash-opt={}'.format(o) for o in
-                    TEST_FLASH_OPTS] +
-                   ['--gdb-port', str(TEST_GDB_PORT),
-                    '--telnet-port', str(TEST_TELNET_PORT),
-                    '--dev-id', TEST_DEV_ID,
-                    '--frequency', str(TEST_FREQUENCY),
-                    '--tool-opt', TEST_TOOL_OPT])
+TEST_ALL_PARAMS = list(itertools.chain(
```

Comment:
Curious what difference this makes compared to a shorter `+` considering you're expanding it to a `list()` anyway? It adds an extra `import`.

Reply:
ping?